index.html

```diff
@@ -496,6 +549,7 @@ <h3><dfn>RTCQuicStreamEvent</dfn></h3>
         [ Constructor (DOMString type, RTCQuicStreamEventInit eventInitDict), Exposed=Window]
 interface RTCQuicStreamEvent : Event {
     readonly        attribute RTCQuicStream stream;
+    readonly        attribute RTCQuicStreamParameters parameters;

```

Comment:
If stream.direction is there (which it is in this PR), we don't need the parameters here, do we?

Reply:
Removed.